Editor/Resources/EditorWindow/GameLiftPlugin.uxml

```diff
@@ -0,0 +1,33 @@
+<ui:UXML xmlns:ui="UnityEngine.UIElements" xmlns:uie="UnityEditor.UIElements" xsi="http://www.w3.org/2001/XMLSchema-instance" engine="UnityEngine.UIElements" editor="UnityEditor.UIElements" noNamespaceSchemaLocation="../../../../../UIElementsSchema/UIElements.xsd" editor-extension-mode="False">
```

Comment:
Dumb question since I don't have too much experience with the Unity editor? 

Is the uxml and uss auto-generated from the components created from the Unity editor? What would be the best way to review this? Could you add in a screenshot of the UX on the PR description to help make reviewing easier? Same for other PRs which include graphical elements

Reply:
Within UI Toolkit in unity, there is a tool UI BUilder which will generate `.uxml` and `.uss` files, however we can also just manually write and edit them.

If screenshots are the preferred way to review the visuals then we can start providing them on any UI affecting PR. 

And if licence text is required on all files, then that shall be added too :)

Reply:
Thanks for providing the screenshots! Yes, license information on all files is a requirement. Everytime you edit these via the editor will the licensing get overwritten? But i guess that's easy to catch during a PR so I'm not too concerned about it.